run.sh

```diff
@@ -294,6 +302,26 @@ workload_1_java() {
                                 -warmup="${WARMUP}"
 }
 
+workload_bidi_java() {
+  java -Dio.grpc.netty.shaded.io.netty.native.workdir=/ -Dorg.conscrypt.native.workdir=/ -jar /usr/bin/java-cli -project="${PROJECT}" \
+                                -bucket="${BUCKET_NAME}" \
+                                -test_type="bidi" \
+                                -api="${API}" \
+                                -object_size="${OBJECT_SIZE}..${OBJECT_SIZE}" \
+                                -workers="${WORKERS}" \
+                                -samples="${SAMPLES}" 
+}
+
+workload_nobidi_java() {
+  java -jar /usr/bin/java-cli -project="${PROJECT}" \
```

Comment:
You need to add the flags to this comment as well; Thinking you may want to use an alias for Java related commands so the flags are always present; but only really have an impact for gRPC.

Reply:
sorry about that miss. added them in. probably right about aliasing them.